packages/phone-number-privacy/combiner/src/index.ts

```diff
@@ -1,31 +1,36 @@
+import { logger } from '@celo/phone-number-privacy-common'
 import * as functions from 'firebase-functions'
-import logger from './common/logger'
 import { VERSION } from './config'
 import { handleGetContactMatches } from './match-making/get-contact-matches'
 import { handleGetBlindedMessageSig } from './signing/get-threshold-signature'
 
+require('dotenv').config()
+
 // DEPRECATED, TODO: Remove once clients are all on contract kit version 4.11
 export const getDistributedBlindedSalt = functions
   .region('us-central1', 'europe-west3')
-  .https.onRequest(async (request, response) => {
-    logger.info('Begin getDistributedBlindedSalt request')
-    return handleGetBlindedMessageSig(request, response)
+  .https.onRequest(async (req, res) => {
```

Comment:
In general, it might be useful to include a sessionId/correlationId in each logging event so that we can connect the events together after the fact. We don't have to do it in this PR but just something to think about.

Reply:
Good idea. Are you thinking something like each request has an id so that we can link all the logs originating from that request together?